src/macaron/config/defaults.py

```diff
@@ -37,24 +37,26 @@ def get_list(
         If ``strip`` is True  (default: True), strings are whitespace-stripped and empty strings
         are removed from the final result.
 
-        If ``duplicated_ok`` is True (default: False), duplicated values are not removed from the final list.
+        If `duplicated_ok` is False, duplicated elements which come after the their first instances will
+        be removed from the list. This operation happens after ``strip`` is handled.
 
+        The order of non-empty elements in the list is preserved.
         The content of each string in the list is not validated and should be handled separately.
 
         Parameters
         ----------
         section : str
             The section in ``defaults.ini``.
-        item : str
-            The item to parse the list.
+        option : str
+            The option whose value will be split into the return list of strings.
```

Comment:
```suggestion
            The option whose values will be split into a list of strings.
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/660/commits/ed2a337ad980489722f351fbac97d879329cd0fa